src/core/Logo/index.js

```diff
@@ -2,15 +2,17 @@
 
 import React from 'react'
 import PropTypes from 'prop-types'
-import { bemClassNames } from '../../utils'
+import { componentStyles } from '../../utils'
 
 import './styles.css'
 import iconBlue from './svg/icon-blue.svg'
 import iconWhite from './svg/icon-white.svg'
 import logoBlue from './svg/logo-blue.svg'
 import logoWhite from './svg/logo-white.svg'
 
-const bem = bemClassNames('logo')
+import './styles.css'
+import styles from './styles.json'
+const myst = componentStyles(styles)
```

Comment:
This function would do what `bemClassNames` does today but with the CSS modules.